PL_2_komentarze.py

```diff
@@ -0,0 +1,2722 @@
+                    self.tree.column(tree_columns[indx], width=ilen)
+    
+    def daneklienta(self, event):
+        self.entryKlient1 = entryKlient1
+        self.entryKlient2 = entryKlient2
+        self.entryKlient3 = entryKlient3
+        item = self.tree.identify_row(event.y)
+        entryKlient1.delete(0, END)
+        entryKlient1.insert(END, self.tree.item(item, "values")[0])
+        entryKlient1.validate()
+        entryKlient2.delete(0, END)
+        entryKlient2.insert(END, self.tree.item(item, "values")[1])
+        entryKlient2.validate()
+        entryKlient3.delete(0, END)
+        entryKlient3.insert(END, self.tree.item(item, "values")[2])
+        entryKlient3.validate()
+        oknoklient.destroy()
+        ladujklientow()
+
+    def bind(self, k, f):
+        return self.tree.bind(k, f)
+
+def znajdzklienta():
+
+    wprowadzony = entryKlient1.get()
+    listaponumerowana = list(enumerate(listaklientow))
+    znalezioneindeksy = [i for i,x in listaponumerowana if wprowadzony.lower() in x.lower()]
+    rezultatyklient = []
+    if not znalezioneindeksy:
+        messagebox.showinfo("Nie znaleziono klienta!", "Nie udało się dopasować wprowadzonej frazy do żadnego z klientów.\nWypełnij pola adresowe ręcznie, a następnie kliknij przycisk 'Dodaj klienta', aby zachować dane na przyszłość.")
+    else:
+        global oknoklient
+        oknoklient = Toplevel(root)
+        oknoklient.iconbitmap(r'bin\tubes.ico')
+        oknoklient.title('Klienci zawierający szukane słowo:')
+        for item in znalezioneindeksy:
+            wartoscadresu1 = wsX.cell(item,1).value
+            key = b'Uf-FryHfa-JXDToRDClFsSrv2h56kzEUCK5imvQngbo='
+            f = Fernet(key)
+            kodowanie1 = wartoscadresu1.encode()
+            odkryte1 = f.decrypt(kodowanie1)
+            wartoscadresu11 = odkryte1.decode('utf-8')
+            wartoscadresu2 = wsX.cell(item,2).value
+            kodowanie2 = wartoscadresu2.encode()
+            odkryte2 = f.decrypt(kodowanie2)
+            wartoscadresu22 = odkryte2.decode('utf-8')
+            rezultatyklient.append((listaklientow[item], wartoscadresu11, wartoscadresu22))
+        
+        Drzewo(oknoklient, rezultatyklient, entryKlient1, entryKlient2, entryKlient3)
+        Tooltip(oknoklient, text='Potwierdź wybór klienta podwójnym kliknięciem')
+    
+        if len(rezultatyklient) == 1:
+            tupelek = rezultatyklient[0]
+            entryKlient1.delete(0, END)
+            entryKlient1.insert(END, tupelek[0])
+            entryKlient1.validate()
+            entryKlient2.delete(0, END)
+            entryKlient2.insert(END, tupelek[1])
+            entryKlient2.validate()
+            entryKlient3.delete(0, END)
+            entryKlient3.insert(END, tupelek[2])
+            entryKlient3.validate()
+            oknoklient.destroy()
+            ladujklientow()
+
+def znajdzklientaenter(event):
+    znajdzklienta()
+
+ludzie = {
```

Comment:
Za bardzo ogólna nazwa słownika, nie wiadomo w sumie co zawiera. Tajemnicą też jest co znaczą te liczby. Generalnie zasada jest taka, że im więcej poświęcasz się czasu na tworzenie ładnego, opisowego kodu, nadawaniu czytelnych nazw zmiennym itepe, tym mniej potem się wkurwiasz jak wrócisz do kodu po kilku miesiącach i już nie pamiętasz o co chodziło.  Nakład pracy na dbanie o takie rzeczy zwraca się zawsze.
Gdzieś przeczytałem że kod należy pisać zrozumiały dla ludzi, bo maszynom to jeden chuj czy zmienna to `summaryCostOfOrder` czy `vvQ8_dupa`, byle się składnia spinała.